saleor/payment/gateways/np_atobarai/errors.py

```diff
@@ -0,0 +1,294 @@
+        "does not contain prohibited characters."
+    )
+
+    # Delivery destination (ZIP code)
+    E0100081 = "Please check if the delivery destination(ZIP code) has been entered."
+    E0100082 = (
+        "Please make sure that the delivery destination(ZIP code) "
+        "does not contain prohibited characters."
+    )
+    E0100137 = (
+        "Please check if the delivery destination(ZIP code) "
+        "is in the correct format.(Example: 1070052, 107 - 0052)"
+    )
+    E0100083 = "Please make sure the delivery destination (ZIP code) and address match."
+
+    # Delivery destination (address)
+    E0100084 = "Please check if the delivery destination(address) has been entered."
+    E0100085 = (
+        "Please make sure that the delivery destination(address) "
+        "does not contain prohibited characters."
+    )
+    E0100086 = (
+        "Please confirm that the delivery destination(address) "
+        "is within 55 full - width characters."
+    )
+
+    # Delivery destination (phone number)
+    E0100087 = (
+        "Please check if the delivery destination(phone number) has been entered."
+    )
+    E0100088 = (
+        "Please make sure that the delivery destination(phone number) "
+        "does not contain prohibited characters."
+    )
+    E0100089 = "Please ensure the phone number format is valid."
+    E0100090 = (
+        "If the 1 st to 3 rd digit of the phone number excluding the hyphen is "
+        "“020”, “050”, “060”, “070”, “0 80”, or “0 90”, please ensure "
+        "that the phone number has a total of 11 digits."
+    )
+    E0100091 = "Please check if the delivery phone number is entered."
+
+
+class TransactionCancellationResultError(Enum):
+    EPRO0101 = "Please confirm that at least one normal transaction is set."
+
+    EPRO0102 = (
+        "Please confirm that 1, 000 or fewer sets of normal transactions are set."
+    )
+
+    EPRO0105 = "Please check if the NP Transaction ID has been entered."
+
+    EPRO0106 = "Please check if the same NP Transaction ID is duplicated."
+
+    E0100002 = (
+        "Please check if the NP Transaction ID is in "
+        "half - width alphanumeric characters."
+    )
+
+    E0100003 = "Please check if the NP Transaction ID is 11 digits."
+
+    E0100113 = "Please confirm that the transaction in question exists."
+
+    EPRO0107 = "Please confirm that the transaction s an NP card transaction."
+
+    E0100114 = "Please confirm that the transaction is prior to customer payment."
+
+    E0100118 = "Please confirm that the transaction is not cancelled."
+
+    E0100131 = "Please confirm that the transaction is prior to returning to merchant."
+
+    E0100132 = "Please confirm that the payment method is as expected."
+
+
+def get_error_messages_from_codes(
+    error_codes: Iterable[str], error_enum_cls: Type[Enum]
+) -> List[str]:
+    error_messages = []
+    for code in error_codes:
+        try:
```

Comment:
Wouldn't be better to instead of having error cls, we could have a dict map? This could drop this try/except as you could use .get(code, default_error)